epos_hardware/src/util/epos.cpp

```diff
@@ -310,292 +393,336 @@ bool Epos::init() {
     VCS_FROM_SINGLE_PARAM_OPTIONAL(safety_nh, int, max_following_error, SetMaxFollowingError);
     VCS_FROM_SINGLE_PARAM_OPTIONAL(safety_nh, int, max_profile_velocity, SetMaxProfileVelocity);
     VCS_FROM_SINGLE_PARAM_OPTIONAL(safety_nh, int, max_acceleration, SetMaxAcceleration);
-    if(max_profile_velocity_set)
+    if (max_profile_velocity_set)
       max_profile_velocity_ = max_profile_velocity;
     else
       max_profile_velocity_ = -1;
   }
```

Comment:
I like all these extra scopes, why not break all these blocks out into functions?